Task1/Task1/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Numerics;
+
+namespace Task1
+{
+    class Program
+    {
+        class MyObject
+        {
+            private Vector2 _position;
+            public readonly string Sprite = "";
```

Comment:
https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods